src/components/TimerContainer.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { publish, subscribe } from './Event';
+
+function TimerContainer(props) {
+  const { questionNum, quiz } = props;
+  const [count, setCount] = useState(Number(quiz.timer));
+
+  const updateCount = () => {
+    if (count > 1) {
+      setCount((lastCount) => lastCount - 1);
```

Comment:
Why do we need lastCount here?
Can we just do `setCount(count - 1)`
If not, why? If yes, why?